.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] [REQUIRED] Good job ignoring the `test.md` file, please also add the node_modules directory (as it can be added by accident while setting up linters). You can add it as
```
node_modules/

```
then it should be ignored by git.